README.md

```diff
@@ -853,27 +1046,27 @@
     }
     ```
 
-  - Use camelCase when naming objects, functions, and instances
+  - Use camelCase when naming objects, functions, and instances.
 
     ```javascript
     // bad
     var OBJEcttsssss = {};
     var this_is_my_object = {};
-    var this-is-my-object = {};
-    function c() {};
+    var o = {};
+    function c() {}
     var u = new user({
       name: 'Bob Parr'
     });
 
     // good
     var thisIsMyObject = {};
-    function thisIsMyFunction () {};
+    function thisIsMyFunction () {}
```

Comment:
Looks like a semicolon was accidentally removed here. Also, there's an errant space in `thisIsMyFunction()`